commands.go

```diff
@@ -3,475 +3,287 @@ package main
 import (
 	"fmt"
 	"sort"
-	"strconv"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/acarl005/stripansi"
-	"github.com/shurcooL/tictactoe"
 )
 
-// runCommands parses a line of raw input from a user and sends a message as
-// required, running any commands the user may have called.
-// It also accepts a boolean indicating if the line of input is from slack, in
-// which case some commands will not be run (such as /tz and /exit)
-func runCommands(line string, u *user, isSlack bool) {
-	if line == "" {
-		return
-	}
-	if u.messaging != nil && !strings.HasPrefix(line, "=") && !strings.HasPrefix(line, "./room") {
-		u.writeln(u.messaging.name+" <- ", line)
-		if u == u.messaging {
-			devbotRespond(u.room, []string{"You must be really lonely, DMing yourself.",
-				"Don't worry, I won't judge :wink:",
-				"srsly?",
-				"what an idiot"}, 30, false)
-			return
-		}
-		u.messaging.writeln(u.name+" -> ", line)
-		return
-	}
+func registerCommands() {
+	commands = append(commands, commandInfo{"clear", "Clears your terminal", clearCommand, false, false, nil})
```

Comment:
add a register function that automatically appends, will cut out a lot of repetitiveness

Reply:
(not high priority)